### ISSUE TITLE
Installation on Moodle 3.5 site hangs

### ISSUE THREAD

**Issue by jleecbd** [catalyst/moodle-auth_saml2#285]
I've both uploaded the zip file and attempted the direct download from the plugins directory.  In both cases the installation gets as far as Validating auth_saml2 then simply stops.  I can install other plugins successfully.  Suggestions?

**Comment by danmarsden** [comment]
HI @jleecbd,

This sounds like a site configuration issue or a problem with the core installer. Unfortunately we are not in a position to provide free installation and configuration support for all our plugins. If you are looking for free support the best place to ask for help is in the community forums on moodle.org - I'd recommend you ask for help in the Installation and upgrading forum here:
https://moodle.org/mod/forum/view.php?id=28

(I'll keep an eye out for your post there just in case someone finds this relates to a bug with our code)

If you are looking for commercial level support feel free to drop us a private message and we can provide information on our rates etc.

also - It might be a good idea to set debugging to developer and display errors = on to see if any other useful information is reported.

Good luck!

**Comment by rpiyaket** [comment]
Hi @jleecbd,
I'm experiencing the same problem. Did you ever find a solution?
Many thanks.

**Comment by jonnyso** [comment]
Hi @rpiyaket 

Managed to fix it. In my case, for some reason it was exeeding the execution time limit for PHP. All I did was to increace the time limit on my php.ini to 120 seconds and ran the installation again.

If this doesn't solve your problem, I suggest enabling the Debug mode and mark the checkbox to Display Debug Messages, then when you run the installation Moodle will spit out wathever is making it hang. More information here: https://docs.moodle.org/37/en/Debugging
